packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -107,6 +116,10 @@ export default class VTable {
                     }
                 });
             });
+            if (this.formatInfo) {
+                this.applyFormat(this.formatInfo);
+                saveTableInfo(this.table, this.formatInfo);

```

Comment:
If formatInfo has the same value with default format, do not save and delete any existing format info from the table